CONTRIBUTORS.md

```diff
@@ -0,0 +1,24 @@
+# Getting Started
+
+Thanks for agreeing to help contribute to the Vaxx Website!
+
+By contributing, you agree to the [code of conduct here](https://github.com/CovidEngine/vaxxnz/blob/main/CODE_OF_CONDUCT.md)
+
+Here are some ways to help:
+
+- Add missing vaccination centres to [our spreadsheet](https://docs.google.com/spreadsheets/d/1dbAFVdtQParU78_j3eR1i5iA6XpzLsrR_WvzTATPK2Y/edit)! Please add as much info as you can find :heart:
+
+- Join the Vaxx Discord and introduce yourself at https://discord.gg/6qcQxf4V
+
+- Once you've joined the Discord, introduce yourself, and let the channel know you would like to contribute in #introductions 🥳
+
+- Have a look at some [open issues](https://github.com/CovidEngine/vaxxnz/issues) and see if anyone needs any help investigating a problem or bug
+
+- [Review our roadmap ](https://github.com/CovidEngine/vaxxnz/projects/2) and join the discussion
+
+- Propose a new idea or feature by [Creating an issue](https://github.com/CovidEngine/vaxxnz/issues)
+
+## For Designers:
```

Comment:
Maybe also have a link to the developers doc?

Reply:
more for nontechnical work